bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/transactions/TransactionResourceManager.java

```diff
@@ -321,8 +335,6 @@ private void invokeCommittedFunction(Strand strand, String transactionId, String
             for (int i = fpValueList.size(); i > 0; i--) {
                 FPValue fp = fpValueList.get(i - 1);
                 //TODO: Replace fp.getFunction().apply
-//                BRuntime.getCurrentRuntime().invokeFunctionPointerAsyncIteratively(fp, 1, () -> args,
-//                        results -> {}, () -> null);
                 fp.getFunction().apply(args);
```

Comment:
Haven't we got fix from Runtime team for this issue.

Reply:
created an issue for this
https://github.com/ballerina-platform/ballerina-lang/issues/26187